### PR TITLE
Front end home page

### PR DESCRIPTION
**Pull request by kdcgovender**

### CODE REVIEW

File: WebLevelup/src/scripts/questions.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+      return question;
+    });
+    questions=formattedQuestions;
+  })
+  .catch(error => {
+    console.error('There was a problem with the fetch operation:', error);
+  });
+
+
+// questions = [
```
Comment (AlexKruger2815) [inline WebLevelup/src/scripts/questions.js]: Can we remove these comments?

File: WebLevelup/src/styles/style.css [hunk 2]
```diff
@@ -0,0 +1,393 @@
...
+    background-size: cover;
+    background-position: center;
+    transition: .3s ease;
+    overflow: hidden;
+    pointer-events: auto;
+}
+
+
+.main.active {
+    filter: blur(15px);
```
Comment (AlexKruger2815) [inline WebLevelup/src/styles/style.css]: You missed the px here I believe

File: WebLevelup/src/scripts/questions.js (hunk 1 above)
Comment (kdcgovender) [reply WebLevelup/src/scripts/questions.js]: yes. I'll remove now

File: WebLevelup/src/styles/style.css (hunk 2 above)
Comment (kdcgovender) [reply WebLevelup/src/styles/style.css]: ahh. Thank you

**Pull request merged**